README.md

```diff
@@ -218,6 +221,46 @@ Options:
 - `bitLength`: Number of id bits. (default: 128)
 - `multibase`: Account for multibase encoding. (default: true)
 
+### `generateKeySeed(options)`
+
+`generateKeySeed()` and `decodeKeySeed()` methods are for creating  and
+decoding key pair seeds which will be used for auto-generating app ids.
+
+`generateKeySeed()` generates a string key seed ID.
+
+```js
+import {generateKeySeed} from 'bnid';
+const keySeed = await generateKeySeed();
+// Example keySeed: z1Aaj5A4UCsdMpXwdYAReXa4bxWYiKJtdAvB1zMzCHtCbtD
+```
+Options:
+- `encoding`: Encoding. (default: `base58`)
+- `bitLength`: Number of id bits. (default: 32 * 8)
+- `multibase`: Account for multibase encoding. (default: true)
+- `multihash`: Account for multihash encoding. (default: true)
+
+### `decodeKeySeed(options)`
+
+Decodes a key seed ID into a 32-bytes array seed bytes.
+
+```js
+import {decodeKeySeed} from 'bnid';
+const keySeed = 'z1Aaj5A4UCsdMpXwdYAReXa4bxWYiKJtdAvB1zMzCHtCbtD';
+decoded = decodeKeySeed({id: keySeed});
```

Comment:
I think conflating secret key seed bytes with an "identifier" is dangerous. We should adjust param names and code to ensure that those two are not mixed. Ultimately, these helper functions are about encoding/decoding secret key seeds for use in identifier generation -- but this library doesn't do the identifier generation, some external library that consumes the seed does (typically, it generates a key pair and uses the public key as the ID or as part of the ID).

The docs should be more clear about this pattern and how this library helps generate secret key bytes and encode them for storage and later generation/regeneration of the same key pair ... *from which* an identifier can be created.

Reply:
This is also different from the use case we have where we want to generate (typically) 16 random bytes for an identifier and encode it using multibase+multihash.